README.md

```diff
@@ -106,6 +106,19 @@ runner.train(
     verbose=True,
     load_best_on_end=True,
 )
+
+# loader evaluation
+metrics = runner.evaluate_loader(
+    model=runner.model, # or any other model, compatable with current runner
+    loader=loaders['valid'],
+    callbacks=[
+    dl.AccuracyCallback(
+        input_key="logits", target_key="targets", topk_args=(1, 3, 5)),
+        # ... any other callback, compatable with runner
+    ]
+    )
+assert 'accuracy' in metrics.keys()
```

Comment:
could you please run `catalyst-make-codestyle`on this block?

Reply:
just want the code to be the same style

Reply:
Sure.) 
I've added a changed by catalyst-codestyle block to index.rst too.